models/instrument.py

```diff
@@ -918,6 +919,46 @@ def get_ra_dec_for_section(self, exposure, section_id):
         self.check_section_id(section_id)
         return None, None
 
+    def get_ra_dec_corners_for_section( self, ra, dec, section_id ):
+        """Get nominal corners of a sensor section given overall array RA and Dec.
+
+        Parameters
+        ----------
+           ra, dec: float
+              Position of the entire array (not of the specific section).
+
+           section_id: str
+
+        Returns
+        -------
+           dict, 10 keys: (ra|dec)_corner_(0|1)(0|1), (min|max)(ra|dec)
+
+        """
+        raise NotImplementedError( f"{cls.__name__} needs to implement get_ra_dec_corners_for_section" )
+
+
+    def get_ra_dec_for_section_of_exposure(self, exposure, section_id):
```

Comment:
The of_exposure is important because you pass an exposure here, whereas get_ra_dec_for_section you just pass an ra and dec and a section_id.  The latter didn't exist previously, and get_ra_dec_for_section required you to pass an exposure.  I needed the ability to get the ra and dec of a section without an exposure to use, which is why I made these changes.

In the jargon of SeeChange, a "section" is what I might have called a "chip", only this way it's less wrong if we use a detector where (say) we get images divided up by amplifier instead of by chip, or some such.